storage/block_devices.go

```diff
@@ -632,6 +660,18 @@ func isBlockDeviceAvailableDeep(blockDevices []*BlockDevice) bool {
 			break
 		}
 
+		// Hack for Logical Volumes
+		// Since lvms do not have Partition Labels, if the Logical Volume name
+		// which matches Advance Labels, force set the Partition Label
+		if bd.Type == BlockDeviceTypeLVM2Volume {
+			log.Debug("isBlockDeviceAvailableDeep: Found lvm name: %s", bd.Name)
+			if matches := lvmAdvancedLabelExp.FindStringSubmatch(bd.Name); len(matches) == 2 {
```

Comment:
Can we LOG an error to disk if we the regex match failed

Reply:
Done:
```
2020/06/23 22:52:17 [DBG] isBlockDeviceAvailableDeep: Found lvm name: VolGrpClearRoot-blah
2020/06/23 22:52:17 [DBG] lvm name VolGrpClearRoot-blah does not match Advanced regex \w+-(CLR_[0-9a-zA-Z-_\+.]+$)

```